jekyll/_includes/snippets/gpu-linux-resource-table.md

```diff
@@ -1,5 +1,9 @@
 Class | vCPUs | RAM | GPUs | GPU model | GPU Memory (GiB) | Disk Size (GiB) | Cloud | Server
 ---|---|---|---|---|---|---|---|---
+gpu.nvidia.small |   4   | 16  | 1 | Nvidia Tesla P4 | 16 | 300 | <i class="fa fa-check" aria-hidden="true"></i> | <i class="fa fa-times" aria-hidden="true"></i>
+gpu.nvidia.small.gen2 |   4   | 16  | 1 | Nvidia A10G | 16 | 300 | <i class="fa fa-check" aria-hidden="true"></i> | <i class="fa fa-times" aria-hidden="true"></i>
```

Comment:
also gpu memory is 24 based on g5 docs: https://aws.amazon.com/ec2/instance-types/g5/